src/v2/Components/ArtworkFilter/__tests__/ArtworkFilterMobileActionSheet.jest.tsx

```diff
@@ -60,7 +60,7 @@ describe("ArtworkFilterMobileActionSheet", () => {
 
     expect(context.stagedFilters).toEqual({
       ...initialArtworkFilterState,
-      reset: true,
+      reset: false,
```

Comment:
This is my only ❓ — this spec seems like it _should_ be resetting this flag. It's been a while since I've looked at this code, so I don't even recall the "why" of this... but if the "Clear all" button doesn't flip this flag to `true`, then what does?

Reply:
You are right, the "Clear button" set `reset: true`. But immediately effect reacted for `reset` changes and updates filters (set `false` value to `reset` flag).

Unfortunately I found the bug with this solution. When user clear all filters "apply" button are disable and it is impossible to apply reset filters. So I try to fix it

Reply:
Fix the bug, so `reset` flat works correctly now